app/scripts/filters/util.js

```diff
@@ -515,4 +515,13 @@ angular.module('openshiftConsole')
     return function(feature) {
       return _.get(Constants, ['ENABLE_TECH_PREVIEW_FEATURE', feature], false);
     };
+  })
+  .filter('hasHTML', function() {
```

Comment:
Let's not try to test for HTML tags because this can have false positives. I would remove this filter.